test_cli.py

```diff
@@ -126,10 +162,25 @@ def run_command(
             kwargs = dict(kwargs, encoding="utf-8")
         if include_stderr:
             kwargs = dict(kwargs, stderr=subprocess.STDOUT)
-        output = subprocess.check_output(
-            [a.encode() if type(a) is str else a for a in args], **kwargs
-        )
-        exitcode = 0
+        if not force_subprocess and isinstance(args, list) and args[0] == cl:
```

Comment:
why would `args` not be a list?

Reply:
Yeah, looks like `args` is always a list. Changing.